pkg/onit/cli/create.go

```diff
@@ -79,5 +79,6 @@ func runCreateClusterCommand(cmd *cobra.Command, _ []string) error {
 	setup.Partitions().Raft()
 	setup.Topo().SetReplicas(1)
 	setup.Config().SetReplicas(1)
+	setup.Gui().SetReplicas(1)
```

Comment:
maybe better to do something similar to cli and have a --set onos-gui.enabled=true instead of having it as default

Reply:
Yep. I was thinking about it.